public/game-master/styles/game-master.styl

```diff
@@ -21,8 +21,15 @@
 		
 		.round-title
 			flex-grow 4
-		.round-timer
-			flex-grow 1
+			line-height 1.2em
```

Comment:
Generally speaking `em`s are bad for component styling, since it prevents free component containment, right?
Why base timer's font-size on its container rather than the base font?